### ISSUE TITLE
Document preprocessor compile flags in Readme.md

### ISSUE THREAD

**Issue by sebawild** [sebawild/cpython#1]
Willem, can you add a quick bullet list of all compile flags you added in the existing project readme (start a new section at the top) that explains what each of these do?

I think it would also be advisable to rename those to use a common prefix, so that they can easily be identified as our additions.  The infamous `WB_` seesm to suggest itself ...

**Comment by WilliamCawleyGelling** [comment]
yep, i can do this now, currently they are a bit all over the place as the combination of three of them is what i currently need for printing everything for processing. I will make add a quick bullet point list but i will also clean up all the flags

**Comment by sebawild** [comment]
Sounds good. Please include (if that applies) if any combination is not possible.
For example, one probably shouldn't include both Timsort and Powersort (although here a single flag suffices that toggles these)
